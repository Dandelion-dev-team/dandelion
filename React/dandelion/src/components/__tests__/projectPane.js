import React from "react"
import renderer from "react-test-renderer"
import ProjectPane from "../projectPane"

describe("Project Card", () => {
  it("renders correctly", () => {
    const testProp = { 'id': '1', 'title': 'Test Title', "start_date": "2022-02-22T00:08:17.028Z", "end_date": "2022-02-22T00:08:17.028Z", "partner": "Test school", "project_image_link": "test link" };
    const tree = renderer
      .create(<ProjectPane dataProp={testProp}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})