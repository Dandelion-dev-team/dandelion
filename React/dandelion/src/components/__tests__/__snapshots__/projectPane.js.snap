// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Card renders correctly 1`] = `
<div>
  <div
    className="panel-content"
  >
    <div
      className="project-title"
    >
      <h3>
        Test Title
         Project
      </h3>
      <h3>
        Tue Feb 22 2022
         -
         
        Tue Feb 22 2022
         
      </h3>
      <h3>
        Test school
      </h3>
      <h3>
        Owner - 
      </h3>
      <div
        className="feat-img"
      >
        <img
          src="test link"
        />
      </div>
      <div
        className="experiment-row"
      >
        <div
          className="experiment-card"
        >
          <div
            className="card-img"
          >
            <img />
          </div>
          <div
            className="title"
          >
            <h2 />
          </div>
          <div
            className="center"
          >
            <div
              class="vl"
            />
          </div>
          <div
            className="description"
          >
            <h2 />
          </div>
        </div>
      </div>
    </div>
    <div
      className="btn-row"
    >
      <button
        className="submitButton"
        id="inv"
      >
        Invite School
      </button>
      <button
        className="submitButton"
      >
        Edit Project
      </button>
      <button
        className="submitButton"
        id="comp"
      >
        Project Complete
      </button>
    </div>
  </div>
</div>
`;
